data/src/main/java/com/bottlerocketstudios/brarchitecture/data/model/SnippetDto.kt

```diff
@@ -8,10 +8,10 @@ import java.time.ZonedDateTime
 
 @JsonClass(generateAdapter = true)
 @Parcelize
-data class Snippet(
+data class SnippetDto(
     @Json(name = "id") val id: String? = null,
     @Json(name = "title") val title: String? = null,
     @Json(name = "is_private") val isPrivate: Boolean? = null,
     @Json(name = "owner") val owner: UserDto? = null,
-    @Json(name = "updated_on") val updated: ZonedDateTime? = null
-) : Parcelable
+    @Json(name = "updated_on") val updated: ZonedDateTime? = null,
+) : Parcelable, Dto
```

Comment:
@VanyaHuamanBottleRocket Thanks for the trailing command and extending `Dto`

Reply:
Yup